arangod/RestHandler/RestAdminLogHandler.cpp

```diff
@@ -113,13 +113,27 @@ RestStatus RestAdminLogHandler::execute() {
       return reportLogs(/*newFormat*/ true);
     } else if (suffixes.size() == 1 && suffixes[0] == "level") {
       handleLogLevel();
+    } else if (suffixes.size() == 1 && suffixes[0] == "structured") {
+      handleLogStructuredParams();
     } else {
       generateError(rest::ResponseCode::BAD,
                     TRI_ERROR_HTTP_SUPERFLUOUS_SUFFICES,
-                    "superfluous suffix, expecting /_admin/log/entries");
+                    "superfluous suffix, expecting /_admin/log/<suffix>, "
+                    "where suffix can be either 'entries', 'level' or 'structured'");
+    }
+  } else if (type == rest::RequestType::PUT && suffixes.size() == 1) {
+    if (!suffixes.empty()) {
+      if (suffixes[0] == "level") {
+        handleLogLevel();
+      } else if (suffixes[0] == "structured") {
+        handleLogStructuredParams();
+      } else {
+        generateError(rest::ResponseCode::BAD,
+                      TRI_ERROR_HTTP_SUPERFLUOUS_SUFFICES,
+                      "superfluous suffix, expecting /_admin/log/<suffix>, "
+                      "where suffix can be either 'level' or 'structured'");
+      }
     }
-  } else {
```

Comment:
This case should be handled. If someone uses a wrong HTTP method or adds extra suffixes to the URL, the server won't react properly.